src/DashboardService.Models.Dto/Responses/ChangeLogResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using LT.DigitalOffice.DashboardService.Models.Dto.Models;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Responses;
+
+public class ChangeLogResponse
```

Comment:
ChangeLogResponse and ChangeLogInfo are identical. Why do you need two models instead of one?

Reply:
They have different purposes. One is for transferring one entity. The second is for passing a list of entities. They do not differ in content

Reply:
Think of it as a consumer. Would you like to get entities by change or otherwise?

I agree with @VLiamin and it not gonna change